cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -68,13 +68,9 @@ def _installing_manager(self):
         return MANAGER_SERVICE in config[SERVICES_TO_INSTALL]
 
     def _deploy_configuration(self):
-        try:
-            ipv6_enabled = bool(
-                socket.getaddrinfo('localhost', SECURE_PORT,
-                                   family=socket.AddressFamily.AF_INET6)
-            )
-        except socket.gaierror:
-            ipv6_enabled = False
+        lo_ip6_addr = sudo(['ip', '-6', 'addr', 'show', 'dev', 'lo'],
+                           ignore_failures=True).aggr_stdout.strip()
```

Comment:
Does this actually need sudo?
Also, why can we not use the socket.getaddrinfo any more?

Reply:
`getaddrinfo` lied (it always reported IPv6 as enabled on `lo`, which was not true in the envs we tested).  `sudo` might not be required indeed, at least not on CentOS 7.7, on RHEL probably I'd have to provide an exact path (`/usr/sbin/ip`) which might be a better solution, or may it not?

Reply:
OK, makes sense- and yeah, let's rip out sudo and make sure to fully qualify the path (since we only support a limited set of OSes anyway for the manager).